node/node.go

```diff
@@ -189,11 +189,8 @@ func (n *Node) initNetworking() error {
 
 	tlsConfig := network.TLSConfig(n.Config.StakingTLSCert)
 
-	// Initialize validator manager and primary network's validator set
-	n.validators = validators.NewDefaultSet(n.Config.NetworkID)
-	if n.validators.Len() == 0 {
-		return fmt.Errorf("no default validator list (network: %d)", n.Config.NetworkID)
-	}
+	// Initialize default validator set for network
```

Comment:
Should the comment include `empty validator set` as `NewSet()` returns an empty set?